runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/client/grpc/GrpcGetDataStreamRequests.java

```diff
@@ -91,18 +110,26 @@ void addToStreamingGetDataRequest(Windmill.StreamingGetDataRequest.Builder build
     }
   }
 
+  /** Represents a batch of queued requests. Methods are thread-safe unless commented otherwise. */
```

Comment:
shoud this be "Methods are **not** thread-safe unless commented otherwise"?

Reply:
done